src/gui/optionsdialog.ui

```diff
@@ -122,8 +122,8 @@
             <rect>
              <x>0</x>
              <y>0</y>
-             <width>504</width>
-             <height>1064</height>
+             <width>529</width>
+             <height>1319</height>
```

Comment:
I'll undo these at the end of the PR because i haven't found an easy way to do that and Designer will just change them back (if i have to change something).